quiz_game/problems1.csv

```diff
@@ -0,0 +1 @@
+What is the best dish?, Butter Rice
```

Comment:
The quiz currently only allows for 1 string input. It won't be able to match "Butter Rice" correctly.

Reply:
Tanks. Fixed. I didn't read the code thoroughly.